src/__stories__/slideshow-story.tsx

```diff
@@ -45,3 +60,48 @@ Default.args = {
     loop: false,
     withBullets: true,
 };
+
+type WithCarouselContextArgs = {numItems: number};
+
+export const WithCarouselContext: StoryComponent<WithCarouselContextArgs> = ({numItems}) => {
+    return (
+        <CarouselContextProvider>
+            <FixedFooterLayout
+                footer={
+                    <ResponsiveLayout>
+                        <Box paddingY={16}>
+                            <CarouselContextConsummer>
```

Comment:
This is an example of use.
Alternatively, you could create a component and use `useCarouselContext()`

Reply:
Only in slideshow story? why not in carousel if its supported? Could be inside utils too

Reply:
I can create an example story using the carousel, yes

Reply:
done